addons/project/models/project.py

```diff
@@ -438,6 +446,17 @@ def copy(self, default=None):
             default['remaining_hours'] = self.planned_hours
         return super(Task, self).copy(default)
 
+    @api.multi
+    def _get_subtask_count(self):
```

Comment:
Please rename it into `_compute_subtask_count` (and check if it's overriden somewhere else).